internal/cmd/alpha/module/manage.go

```diff
@@ -59,5 +59,8 @@ func runManage(cfg *manageConfig) clierror.Error {
 	if err != nil {
 		return clierror.Wrap(err, clierror.New("failed to check module state"))
 	}
+
+	fmt.Printf("Module %s set to managed\n\n", cfg.module)
```

Comment:
I would suggest using a single new line char to make output consistent with other commands

```suggestion
	fmt.Printf("Module %s set to managed\n", cfg.module)
```